src/hooks/useFilterByCollection.js

```diff
@@ -28,27 +24,13 @@ export const useFilterByCollection = ({
   );
 
   useEffect(() => {
+    if (query.collection === "All" || !query.collection) {
+      return setGallery(entries);
+    }
     if (query.collection) {
```

Comment:
shouldnt this be an else if? I would think that if there are 2 ifs, they both run.

Reply:
there's a return statement so it skips the last if statement